cdc/sinkv2/eventsink/txn/mysql/mysql.go

```diff
@@ -191,6 +200,167 @@ type preparedDMLs struct {
 	rowCount  int
 }
 
+// convert2RowChanges is a helper function that convert the row change representation
+// of CDC into a general one.
+func convert2RowChanges(row *model.RowChangedEvent, tableInfo *timodel.TableInfo) *sqlmodel.RowChange {
+	preValues := make([]interface{}, 0, len(row.PreColumns))
+	for _, col := range row.PreColumns {
+		if col == nil {
+			// will not use this value, just append a dummy value
+			preValues = append(preValues, "omitted value")
+			continue
+		}
+		preValues = append(preValues, col.Value)
+	}
+
+	postValues := make([]interface{}, 0, len(row.Columns))
+	for _, col := range row.Columns {
+		if col == nil {
+			postValues = append(postValues, "omitted value")
+			continue
+		}
+		postValues = append(postValues, col.Value)
+	}
+
+	return sqlmodel.NewRowChange(
+		row.Table,
+		nil,
+		preValues,
+		postValues,
+		tableInfo,
+		nil, nil)
+}
+
+func convertBinaryToString(row *model.RowChangedEvent) {
+	for i, col := range row.Columns {
+		if col == nil {
+			continue
+		}
+		if col.Charset != "" && col.Charset != charset.CharsetBin {
+			colValBytes, ok := col.Value.([]byte)
+			if ok {
+				row.Columns[i].Value = string(colValBytes)
+			}
+		}
+	}
+}
+
+// 对 delete event 在这里做一些特殊判断 如 id<? and id>? 这种条件的sql语句
```

Comment:
pls write the comments in English

Reply:
ok, I will refine all comments after all integration test cases are passed.